dracut/anaconda-lib.sh

```diff
@@ -101,6 +101,46 @@ anaconda_live_root_dir() {
     anaconda_mount_sysroot $img
 }
 
+anaconda_net_root() {
+    local repo="$1"
+    info "anaconda: fetching stage2 from $repo"
+
+    # Try to get the local path to stage2 from treeinfo.
+    treeinfo=$(fetch_url $repo/.treeinfo 2> /tmp/treeinfo_err) && \
+        stage2=$(config_get stage2 mainimage < $treeinfo)
+
+    # No treeinfo available.
+    [ -z "$treeinfo" ] && debug_msg $(cat /tmp/treeinfo_err)
+
+    # Use the default local path to stage2.
+    if [ -z "$treeinfo" -o -z "$stage2" ]; then
+        warn "can't find installer mainimage path in .treeinfo"
```

Comment:
I have a feeling that `mainimage` is typo and I guess we can fix it now. What do you think?

Reply:
Fixed.